app/serializers/insecure_user_serializer.rb

```diff
@@ -0,0 +1,7 @@
+class InsecureUserSerializer < ActiveModel::Serializer
```

Comment:
That is a good question.  Did you create a migration that specified `InsecureUser`?  If not, you are good to go!

For the record, these serializers are purposely abstracted from what's in the database.  The model represents what is in the database, and the serializer is an alternative representation of the model